app/views/themes/reshare_show/_user_util_links.html.erb

```diff
@@ -0,0 +1,24 @@
+<ul id="user_utility_links" class="nav navbar-nav navbar-right">
+  <%= render 'shared/locale_picker' if available_translations.size > 1 %>
+  <% if user_signed_in? %>
```

Comment:
we only want admins to see this. other users will be signed in, but we don't want them to see anything except for the viewer + metadata.

Reply:
Ok sounds good, i just figured the `user_signed_in?` would not apply to them unless they were signed in with the devise (which is different from sso i thought?). I can change it to only admin users though

Reply:
you will be able to sign in both ways, the SSO is still through devise.